### PR TITLE
Adding Mmd, Apointments category and stability assessment

### PR DESCRIPTION
**Pull request by Marymary-dev**
https://thepalladiumgroup.atlassian.net/browse/KHP3-5391

### CODE REVIEW

File: Scripts/REPORTING/1_load_Linelist-FACTART.sql [hunk 1]
```diff
@@ -94,6 +94,13 @@ Select distinct
     ART.ScreenedForDepression,
     case when ncd.[Mental illness] is null then 0 else ncd.[Mental illness] end as HasMentalIllness,
     case when ncd.Dyslipidemia is null then 0 else ncd.Dyslipidemia end as HasDyslipidemia,
+    onMMD,
+    StabilityAssessment,
+    Case When DATEDIFF(DAY,LastVisitDate,NextAppointmentDate) <=89 THEN '<3 Months'
+    when DATEDIFF(DAY,LastVisitDate,NextAppointmentDate) >=90 and DATEDIFF(DAY,LastVisitDate,NextAppointmentDate) <=150 THEN '<3-5 Months'
+    When DATEDIFF(DAY,LastVisitDate,NextAppointmentDate) >151 THEN '>6+ Months'
+    Else 'Unclassified'
+    END As AppointmentsCategory,
```
Comment (nobert-mumo) [inline Scripts/REPORTING/1_load_Linelist-FACTART.sql:103]: @Marymary-dev would it make sense to move this one level to the fact table. And then update all the reporting tables to pick the column from the fact?

Comment (Marymary-dev) [reply Scripts/REPORTING/1_load_Linelist-FACTART.sql]: yes @nobert-mumo  precisely, this has been corrected

**Comment by Marymary-dev** [comment]
The following variables have been added: MMD, Appointments Category and Stability assessment

https://github.com/palladiumkenya/dwh-etl/pull/401#issue-2222323509

**Review by nobert-mumo — COMMENTED** [review]
LGTM!

**Review by nobert-mumo — APPROVED** [review]

File: Scripts/REPORTING/1_load_Linelist-FACTART.sql (hunk 1 above)
Comment (nobert-mumo) [reply Scripts/REPORTING/1_load_Linelist-FACTART.sql]: @Marymary-dev just rem if there is another aggregate table utilizing the same columns, we can now read differently from the fact just to maintain one source of the logic

**Pull request merged**